src/platform/common/platform/fs-paths.ts

```diff
@@ -1,5 +1,5 @@
 import { Uri, WorkspaceFolder } from 'vscode';
-import * as path from 'path-browserify';
```

Comment:
We don't need to force the use of `path-browserify` in webpack we can swtich the `path` to `path-browserify`.
Note: This has already been done, hence this makes it easier to use this file (code) with both node & desktop code.

Reply:
This also fixes the failures in the tests

Reply:
I disagree. This is making it harder to determine what code works in the browser and which doesn't. I think the better change here is to switch fs-paths.ts to fs-paths.node.ts and remove the 'path' allow from the linter.

Reply:
Oh actually I've already split them in the main extension. What was the test failure then? I think it's likely we need another linter rule to detect usage of uri.fsPath as that should only be allowed in '.node.ts' files.

Reply:
Ah this is the test failure:
https://github.com/microsoft/vscode-jupyter/runs/5815274707?check_suite_focus=true

I believe this can be fixed by importing from the common fs-paths instead of the fs-paths.node.ts in the test that's failing.

Reply:
I think we should discuss this, having `path` and `path-browserify` complicates things, this is exactly why we have aliases in webpack, so we can replace things with what we need. here we're duplicating everything, and thats a lot of work.

Reply:
I agree, we should talk about this in a meeting. 

I see a number of options:
- We use webpack to figure out which path we use. This has the downside of you not realizing which one is in use when you write the code (and you make assumptions that it will work in web)
- We write our own path that works like node no matter what (it supports windows still separators)
- We have web/node versions of common things. You have to be explicit about which one you're using. This has the downside of having duplicate code